Formula/logcli.rb

```diff
@@ -44,7 +43,6 @@ def install
     fork { exec Formula["loki"].bin/"loki", "-config.file=loki-local-config.yaml" }
     sleep 3
 
-    output = shell_output("#{bin}/logcli --addr=http://localhost:#{port} labels")
-    assert_match "__name__", output
```

Comment:
`__name__` was removed from this endpoint, see https://github.com/grafana/loki/pull/6229